src/test/java/duck/task/TodoTaskTest.java

```diff
@@ -0,0 +1,38 @@
+package duck.task;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class TodoTaskTest {
+    @Test
+    public void stringify_success() {
+        TodoTask task = new TodoTask("do laundry", false);
+        assertEquals("T010/do laundry", task.stringify());
+
```

Comment:
Perhaps you could set up a constant variable when naming "do laundry, finish hw etc. as these may be considered to be magic strings.